backend/src/utils/permissionNames.ts

```diff
@@ -48,4 +48,8 @@ export const PERMISSION_NAMES = {
   UseExternalSounds: "Use External Sounds",
   UseSoundboard: "Use Soundboard",
   ViewCreatorMonetizationAnalytics: "View Creator Monetization Analytics",
+  SendPolls: "Allows Sending Polls",
+  CreateEvents: "Allows for creating scheduled events, and editing and deleting those created by the current user.",
+  CreateGuildExpressions:
+    "Allows for creating emojis, stickers, and soundboard sounds, and editing and deleting those created by the current user.",
```

Comment:
This should match what is shown in the role permissions tab in the discord client
Also, any chance you can sort this object alphabetically?
```suggestion
  SendPolls: "Create Polls",
  CreateEvents: "Create Events",
  CreateGuildExpressions: "Create Expressions",
```